components/Hero.tsx

```diff
@@ -18,12 +18,10 @@ export default function Hero() {
               className="text-5xl md:text-6xl lg:text-7xl font-semibold 
               tracking-tight text-white-500 leading-tight"
             >
-              Fund Open Source in{" "}<br className="hidden sm:inline" />
+              Find+Fund <br className="hidden sm:inline" />
```

Comment:
I like the original better. It looks cleaner and i think the core message is delivered. I also think the view is less pleasant:

<img width="1483" alt="Screenshot 2024-12-16 at 08 25 09" src="https://github.com/user-attachments/assets/d152759c-41ce-4acd-b76b-7d4e9ba8074b" />

Reply:
ok, let's keep it without the Find!